.github/workflows/benchmark.yml

```diff
@@ -22,6 +22,12 @@ concurrency:
 jobs:
   benchmark:
     runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ github.token }}
+    strategy:
+      matrix:
+        python-version: ['3.9']
+        group: [1, 2]
```

Comment:
im not sure if we want to split the benchmarks, since they need to run on the same hardware for it to be a meaningful comparison

Reply:
When we split master and latest PR still runs on the same hardware. The only difference is some portion of the benchmarks runs on the second job. I am assuming pytest-split always splits the same tests. SO, compile and jac tests may run on different hardware but the same test for aster and PR runs on the same job and thus hardware